### DISCUSSION TITLE
[2022년 9월 30일] : 1부 함께 달인이 되는 비결까지 회고록

### DISCUSSION THREAD

**Discussion by WhorideChicken** [ReadingKing/ReadingThinkingSharing#11] [Show and tell]
## DH
- 프로젝트 주기에 맞게 회고록이나 이슈를 남겨보자
- 아예 처음 배우는 느낌보다는 그동안 학습 된 방식을 활요하면 새로운 영역에서도 적용이 가능할것이다.

- 알고리즘 테스트는 사람을 뽑는것에 있어서는 중요하지는 않는것 같다.
- 사람을 뽑을 때 어떤 프로세스를 거치는지도 좋은 회사를 판가름하는 척도 같다.
- 성장의 계기는 함께 했던 동료분들이 컸다.
— 기술적인 요소 외에도 개발자의 태도를 배우게 되었다.(장인 정신 : 본인의 코드에 책임을 져라)

- 소프트웨어 공학은 10개의 분야로 나뉜다.
- 개발자읭 소프트웨어 공학 분야로 따지면 1/5도 안되는 영역이다.
— 그러는 차원에서 아직 개발자에 불과한 단계이기 떄문에 더욱 노력하고자한다.

- 피드백 요청을 많이 했었다. 그때마다 적절한 답을 들은 경험이 있다.
— 내가 피드백을 했다라는 요청보다는 내가 요청을 받았을 때 그정도 경지가 되도록 노력해야겠다 라고 느꼈다.

- 코드 리뷰를 해주는 면접이 인상적이었다.(면접자도 고민을 한 흔적을 볼 수 있어서 좋았다.)

- 지원자가 지원하는 히스토리를 알 수 있다. 
— 한 번 떨어지면 떨어진 이유를 한 번 더 보게 되기 때문에 신중하게 준비를 해서 지원하자

- git 
 — add single comment : 바로 알림이갑니다.
 — Start Review : Pending 상태가 되기 때문에 웹을 껐다켜도 상태가 살아있다.

App 에서도 똑같은 설정을 할 수 있다. : 브라우저와 모바일간의 연동은 안된다.

## HY
- (더 열심히 정리하겠습니다.)
- 자기계발
 — DH : 출퇴근 시간을 투자해서 개발 유튜브를 봤다.
 	     퇴근 후에는 알고리즘 등의 부족한 부분의 강의를 봤다.
 — MH : 인간관계 상처를 많이 받았다.
             토이 프로젝트를 통해 자기계발을 해왔다.

- 개발을 하며 받는 피드백들 및 해야할일 txt를 보며 고민해보며 찾아보는 시간이 성장하는 시간이라고 느낀다.

- 옆자리 MH 님과도 협업하며 배우는 방법이 있는지 경험 할 수 있어 좋았습니다.
- 주위 팀원에게 도움을 줄 수 있는 방법을 고민하고 있습니다.

- 프로그래머와 디벨롭퍼(개발자)의 차이에서 이분적인 표현인거 같다.
-  기능을 구현을 하면서 설계나 예외 상황을 고려하고 의식하는 행동들이 소프트 웨어 개발자라고 생각한다.
- 컴퓨터 프로그래머는 프로그래밍을 잘 모를 때 기능만 개발하는 사람이라 생각한다.
- Google Design SPRINT를 기억하라

## MH
- 성실함과 꼼꼼함을 중요시한다.
-  프로젝트를 진행하면서 “자랐다”의 느낌을 느끼지만 함께 자랐는지는 아직은 잘 모르겠다.

- 소프트웨어 개발자와 프로그래머의 차이
   — 프로그래머는 효율이 좋지 않더라도 일단 개발을 하지만
   — 소프트웨어 개발자는 프로그래머와 결과는 같지만 더욱 효율적인 구조를 고민 하여 개발하는 사람이라 생각합니다.

- 업무를 더 달라

<img width="1413" alt="스크린샷 2022-09-30 오후 10 35 35" src="https://user-images.githubusercontent.com/97941811/198673532-4746964f-0f97-4e5c-bf6e-95432f98f1ed.png">

**Comment by Joseph-Cha** [comment]
와 먼가 미팅을 하면서 나눴던 생각을 정리할 수 있어서 너무 좋네요 ㅎㅎ
고마워요☺️
